parquet/src/file/metadata/mod.rs

```diff
@@ -584,6 +585,11 @@ impl RowGroupMetaData {
         self.num_rows
     }
 
+    /// Returns the first row number in this row group.
```

Comment:
```suggestion
    /// Returns the global index number for the first row in this row group.
```
And perhaps use `first_row_index` instead? That may be clearer.

Reply:
Agree. Updated.